docs/software/backpack-telemetry.md

```diff
@@ -0,0 +1,70 @@
+
+## Prerequisites
+- ELRS transmitter and receiver with the latest firmware.
+- Backpack-capable receiver (e.g., in the Sentinel Antenna Tracker).
+- Configured ELRS LUA script on your transmitter.
+
+## Setting Up the Backpack Telemetry Feature
+
+### Step 1: Update Firmware
+Ensure both your transmitter and receiver are running the latest firmware. Follow the [firmware update guide](https://www.expresslrs.org/software/quick-start/) for detailed instructions. This feature is available in v3.4.0 onwards.
+
+### Step 2: Enable Backpack Telemetry in ELRS LUA Script
+1. **Access the LUA Script:**
+   - Power on your transmitter and navigate to the System menu.
+   - Execute the ELRS LUA script.
+
+2. **Navigate to the Backpack Section:**
+   - Scroll to find the "Backpack" section within the LUA script.
+   - Select the "Telemetry" option.
+
+3. **Enable Backpack Telemetry:**
+   - Set the "Telemetry" option to `On`.
+   - Exit the script.
+
+4. **Disabling Backpack Telemetry:**
+Backpack telemetry is transmitted on the 2.4G WiFi band, via ESPNOW (this should have a very limited impact on any LoRa signals, as it is modulated via OFDM like a normal WiFi hotspot). Every telemetry message that your transmitter receives from the craft will be re-transmitted via ESPNOW. You may want to disable Backpack Telemetry to reduce your footprint on the RF noise floor when you aren't using it, especially if you are at something like a race day, where there is often contention on the RF spectrum. To disable Backpack Telemetry:
+   - Execute the ELRS LUA script.
+   - Scroll to find the "Backpack" section within the LUA script.
+   - Select the "Telemetry" option.
+   - Set the "Telemetry" option to `Off`.
+   - Exit the script.
+
+### Step 3: Bind Backpack Receivers
+Ensure your Backpack receivers (e.g., the one in the Sentinel Antenna Tracker) are bound using your bind phrase. This allows them to receive telemetry data sent via ESPNOW. See https://github.com/aat-sentinel/Documentation/blob/main/Sentinel%20AAT%20lite%20User%20Guide%20-%20ELRS%20setup.pdf
+
+## Using the Telemetry Feature with Sentinel Antenna Tracker
+
+### Sentinel Configuration
+The Sentinel Antenna Tracker includes an ESP12 Backpack-capable receiver that can consume ESPNOW telemetry frames. Ensure the tracker is properly configured to subscribe to ESPNOW telemetry packets. See https://github.com/aat-sentinel/Documentation/blob/main/Sentinel%20AAT%20lite%20User%20Guide%20-%20ELRS%20setup.pdf
+
+### Viewing Telemetry Data
+Once configured, power up your GPS-equipped craft and establish the link between your ELRS Transmitter and Receiver. Ensure that GPS telemetry is being received on your Handset (go to the Model menu in EdgeTX, and inspect the Telemetry page to make sure you are getting GPS lat lon). The Sentinel Antenna Tracker will be able to read GPS data and other telemetry information from the telemetry feed provided by the ELRS transmitter.
```

Comment:
"lat lon" is triggering spell check errors.

I suggest using "coordinates" instead